bigfastapi/schemas/notification_schemas.py

```diff
@@ -1,17 +1,17 @@
 from datetime import datetime
 import pydantic as pydantic
-from pydantic import Field
+from pydantic import Field, EmailStr
 from typing import Optional, List
 from fastapi import Depends
 
 class NotificationBase(pydantic.BaseModel):
     content: str
-    recipient: str
+    recipient: EmailStr
```

Comment:
A notification can be in-app, it may not be related to email at all. It could be to mobile app, to email, as a menu item...

Reply:
So the creator and recipient should not have an EmailStr type?

Reply:
Okay